src/Tests/TableTestState.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace RonasIT\Support\Tests;
+
+use Illuminate\Support\Facades\DB;
+
+class TableTestState extends BaseTestState
+{
+    public function __construct(string $tableName, array $jsonFields = [])
+    {
+        parent::__construct(
+            tableName: $tableName,
+            jsonFields: $jsonFields,
+            connectionName: DB::getDefaultConnection(),
+        );
+    }
```

Comment:
```suggestion
    public function __construct(
        string $tableName,
        array $jsonFields = [],
        string $connectionName = null,
    ) {
        parent::__construct(
            tableName: $tableName,
            jsonFields: $jsonFields,
            connectionName: $connectionName ?? DB::getDefaultConnection(),
        );
    }
```

`connectionName` must be an argument in the constructor. In this case, we can specify any connection we need.